buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/BuildForkPointDistribution.java

```diff
@@ -121,24 +118,32 @@ private File getGradleCloneTmpDir() {
 
     private void tryBuildDistribution(String commit) throws Exception {
         exec(getGradleCloneTmpDir(), "git", "checkout", commit, "--force");
-        exec(getGradleCloneTmpDir(), (Object[]) getBuildCommands());
+        cleanDirectory(getForkPointDistributionDir().getAsFile().get());
 
-        FileUtils.cleanDirectory(getForkPointDistributionDir().getAsFile().get());
         String baseVersion = new String(Files.readAllBytes(getGradleCloneTmpDir().toPath().resolve("version.txt"))).trim();
 
-        copyToDistributionDir("build/distributions/gradle-" + baseVersion + "-bin.zip", "gradle-" + baseVersion + "-commit-" + commit + "-bin.zip");
-        copyToDistributionDir("subprojects/tooling-api/build/shaded-jar/gradle-tooling-api-shaded-" + baseVersion + ".jar", "gradle-tooling-api-" + baseVersion + "-commit-" + commit + ".jar");
+        exec(getGradleCloneTmpDir(), (Object[]) getBuildCommands(commit, baseVersion));
+        setBaselineVersion(baseVersion);
     }
 
-    private String[] getBuildCommands() {
+    private Object[] getBuildCommands(String commit, String baseVersion) {
+        String[] commands = new String[]{
+            "./gradlew",
+            "install",
```

Comment:
Use `:install`. `install` will also execute the tasks for installing in the local maven repo.

Reply:
Done.